News/Mapping/Resolvers/CommentEntityIdResolver.cs

```diff
@@ -0,0 +1,13 @@
+﻿using News.Entities.Models;
+using News.Entities;
+using AutoMapper;
+
+namespace News.Mapping.Resolvers;
+
+internal class CommentEntityIdResolver : IValueResolver<CommentCreateModel, CommentEntity, Guid>
+{
+    public Guid Resolve(CommentCreateModel source, CommentEntity destination, Guid destMember, ResolutionContext context)
+    {
+        return Guid.NewGuid();
```

Comment:
we don't need to create resolver for such simple action. use mapFrom(_ => new Guid